css-color-adjust-1/style.css

```diff
@@ -0,0 +1,24 @@
+.swatch {
```

Comment:
Credit to the CSS Color spec - I reused some of these styles to create the system color swatches.